src/components/FilterPanel/FilterPanel.docs.mdx

```diff
@@ -0,0 +1,42 @@
+
+import { FilterPanel } from './FilterPanel';
+import { DEFAULT_FILTERS } from './FilterPanel.constants';
+
+# FilterPanel
+
+El componente `FilterPanel` permite seleccionar un filtro entre varias opciones, mostrando el estado seleccionado y disparando un callback al cambiar.
+
+## Ejemplo básico
+
+```tsx
+<FilterPanel
+  filters={DEFAULT_FILTERS}
+  selectedValue="all"
+  onSelect={(value) => alert(`Seleccionado: ${value}`)}
+/>
+```
+
+## Props
+
+| Prop           | Tipo                                      | Descripción                                 |
```

Comment:
The props table documents a `size` prop (`'medium' | 'large' | 'xlarge'`) but the FilterPanel types (FilterPanel.types.ts) do not include a `size` prop. The docs and types are inconsistent.

**Sugerencia:**
```typescript
Either remove `size` from docs or add `size` to the FilterPanelProps type and implement it in the component and styles. Example type change (FilterPanel.types.ts):

export interface FilterPanelProps {
  ...
  size?: 'medium' | 'large' | 'xlarge';
}

And update styles to consume that prop.
```